docs/beaver-builder/about-this-release.md

```diff
@@ -2,54 +2,133 @@
+## Improvements in the wp-admin UI
 
-## Accessibility improvements
+### Duplicate option in post and page lists
 
-The following accessibility changes and improvements have been made:
+For posts (including pages and custom post types) that use a Beaver Builder layout, there's now an **Add Duplicate** option, allowing you to duplicate Beaver Builder layouts to new posts and pages without the need to open the Tools menu in the editor.
 
-* Added a **Screen reader text** field to all modules that allow icon selection, which assists screen readers in describing the icon correctly. See the article about [Icon and Icon Group modules](/beaver-builder/layouts/modules/icon-and-icon-group.md#icon-module) for an example of the HTML output.
-* Added labels to the [Subscribe Form module](/beaver-builder/layouts/modules/subscribe-form/subscribe-form.md#accessibility).
-* [Contact Form module](/beaver-builder/layouts/modules/contact-form/contact-form.md): Added an `aria-describedby` attribute to each input field to assist screenreaders in describing the purpose of the field. Allow both the spacebar and Enter keys to submit the form.
-* Removed blank lines from the [Callout Module](/beaver-builder/layouts/modules/callout-and-call-to-action.md).
-* Testimonials Module: Controls were flipped so the markup matches the display.
+![Duplicate post, a 2.5 feature](/img/2-5-features-duplicate-post.png) 
 
 ## Other improvements
 
-* New **Embedded code** option in row backgrounds.  
-  This features lets you embed HTML and shortcode. See [the article about working with rows](/beaver-builder/layouts/rows/work-with-rows.md#background-section).
-* Added `<main>` to the list of row/column/module container tag options on the **Advanced** tab.  
-The **Advanced** tab is documented [here](/beaver-builder/layouts/advanced-tab-rows-columns-modules.md).
-* Added SpinupWP to the [cache clearing tool](/beaver-builder/troubleshooting/debugging/cache-clearing-tool.md).
+* There is basic support for the official [Font Awesome Plugin](https://wordpress.org/plugins/font-awesome/) so you can use that plugin's shortcodes in your Beaver Builder layouts. If you want to use Font Awesome Pro icons, be sure to follow the plugin installation instructions for enabling your domain in the Font Awesome CDN.
+* There's now an easy way to disable [smooth scrolling](/beaver-builder/advanced-builder-techniques/smooth-scrolling-links.md) so a link jumps to its target on the same page.  
+on the **Advanced** tab where you set the anchor in the **ID** field, add `fl-no-scroll` to the **Class** field  below it. See the [smooth scrolling article](/beaver-builder/advanced-builder-techniques/smooth-scrolling-links.md) for more information about creating target anchors on a page and linking to them.
+* Change in Debug Mode operation: errors are output only if  the`showerrors` parameter is in the URL.  
```

Comment:
This debug item is written in developerese. Does the user have to add this parameter to show errors--or remove the parameter to not show errors? Or is this just offering a reason why they might not see errors when they expect to? Why is this an improvement over how it worked before?

Reply:
Works in the same way as `nohistory` meaning after enabling BB debug mode then in your site URL you add `/?showerrors`

If the URL already has `?` being used such as when using the BB debug URL i.e:

```markup
mysite.com?fldebug=ed52f356
```

Then you need to use `&showerrors` like so:

```markup
mysite.com?fldebug=ed52f356&showerrors
```